### ISSUE TITLE
Can we get the audioSessionId?

### ISSUE THREAD

**Issue by UwaisWisitech** [rxlabz/audioplayer#142]
I want to retrieve audioSessionId, how can I get that?

**Comment by moda20** [comment]
on what platform you are trying to do this?

**Comment by UwaisWisitech** [comment]
I want it on android and iOS both.

**Comment by moda20** [comment]
I don't think this repository implements mediaSession at all, at least on android i am pretty sure.
And what are you doing with it exactly? I did extend an older version of this repo to include mediaSessions and audio focus, if you are interested in that. Otherwise, you would have to code it yourself.

**Comment by UwaisWisitech** [comment]
I want to implement custom equaliser on playing audio, for that I need
audioSessionId.

**Comment by moda20** [comment]
Like I said this repository, does not implement mediaSession in the firstPlace. So you can't get a sessionID with it.
